### PR TITLE
Full screen captions banner api 

### PR DESCRIPTION
**Pull request by PorterNan**
# What

Full screen captions banner api (MTC project requirement), this will be only enabled when people set captionsBanner: {height: full}

image

image

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/CaptionsBanner.tsx [hunk 1]
```diff
@@ -50,16 +52,30 @@ export interface _CaptionsBannerProps {
    * @defaultValue 'default'
    */
   formFactor?: 'default' | 'compact';
+  captionsOptions?: {
+    height: 'full' | 'default';
+  };
 }
 
+const scrollOffsetAllowance = 20;
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/CaptionsBanner.tsx:60]: Change to a constant. 
Can't recall if we have a constant file as well but if not we can keep here but at the top of the file.
```suggestion
const SCROLL_OFF_SET_ALLOWANCE = 20;
```

Comment (PorterNan) [reply packages/react-components/src/components/CaptionsBanner.tsx]: Nice catch! There is constant file, but not in this project

**Review by edwardlee-msft — APPROVED** [review]

File: packages/react-components/src/components/CaptionsBanner.tsx [hunk 2]
```diff
@@ -50,16 +52,30 @@ export interface _CaptionsBannerProps {
    * @defaultValue 'default'
    */
   formFactor?: 'default' | 'compact';
+  captionsOptions?: {
+    height: 'full' | 'default';
+  };
 }
 
+const SCROLL_OFF_SET_ALLOWANCE = 20;
```
Comment (dmceachernmsft) [inline packages/react-components/src/components/CaptionsBanner.tsx:60]: What is this offset for?

**Review by dmceachernmsft — APPROVED** [review]

File: packages/react-components/src/components/CaptionsBanner.tsx (hunk 2 above)
Comment (PorterNan) [reply packages/react-components/src/components/CaptionsBanner.tsx:60]: Interesting, what I can still pass the build, LOL, lemme fix this

**Pull request merged**